src/ts2lang-app.ts

```diff
@@ -18,4 +18,6 @@ export function run() {
         .parse(process.argv);
 
     main(program["file"]);
-}
\ No newline at end of file
+}
+
+if (!module.parent) { run(); }
```

Comment:
This means that if this module is the entry point to the application (`module.parent === null`), then we'll just call `run` directly.
It was added so vscode could start directly from this module.